services/order.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[str] = None
+) -> Order:
+    user = User.objects.get(username=username)
+    if date is not None:
+        order = Order.objects.create(user=user)
+        order.created_at = date
+        order.save()
+    else:
+        order = Order.objects.create(user=user)
```

Comment:
order = Order.objects.create(user=user) is code duplicate move it before if statement